contribute/developer-guide.md

```diff
@@ -76,7 +76,10 @@ When you log in for the first time, Grafana asks you to change your password.
 
 #### Building on Windows
 
-The Grafana backend includes Sqlite3 which requires GCC to compile. So in order to compile Grafana on Windows you need to install GCC. We recommend [TDM-GCC](http://tdm-gcc.tdragon.net/download).
+The Grafana backend includes SQLite which requires GCC to compile. So in order to compile Grafana on Windows you need to install GCC. We recommend [TDM-GCC](http://tdm-gcc.tdragon.net/download). Eventually, if you use [Scoop](https://scoop.sh), you can install GCC through that.
+
+In order to use the `make` command above, you need to install [Make for Windows](http://gnuwin32.sourceforge.net/packages/make.htm) and use it in a Unix shell (f.ex. Git Bash). 
+If you don't want to install these and just want to build the back-end, you can do it as follows: `go run build.go build`. The Grafana binaries will be in bin\\windows-amd64.
```

Comment:
```suggestion
You can simply build the back-end as follows: `go run build.go build`. The Grafana binaries will be in bin\\windows-amd64.
Alternately, if you wish to use the `make` command, install [Make for Windows](http://gnuwin32.sourceforge.net/packages/make.htm) and use it in a Unix shell (f.ex. Git Bash). 
```
I like @achatterjee-grafana suggestion, what about something like above? We don't recommend to run the `make` command but provide information if the user wants to do so.

Reply:
@AgnesToulet thanks, this edit makes sense to me. Applied.